### ISSUE TITLE
[SearchBox] WCAG2AA - searchBox embedded input component errors

### ISSUE THREAD

**Issue by JoaoEnesGoncalves** [lumada-design/hv-uikit-react#782]
**Describe the bug**
searchBox embedded input component errors on inputText label and inputText name

**To Reproduce**

1.  run 
```
node C:\Git\hv-uikit-react\automation\robot\storybook\_resources/accessibility.js --url=http://localhost:9002/iframe.html?id=coresearchbox--nosuggestionsearchbox --standard=WCAG2AA --element="#root>div.Component-content-4" --actions="wait for element input to be visible"
```

https://pentaho.github.io/hv-uikit-react/?path=/story/components--searchbox

**Screenshots**
```
{
  documentTitle: 'Storybook',
  pageUrl: 'http://localhost:9002/iframe.html?id=coresearchbox--simplesearchbox',
  issues: [{
      code: 'label',
      type: 'error',
      typeCode: 1,
      message: 'Form elements must have labels (https://dequeuniversity.com/rules/axe/3.4/label?application=axeAPI)',
      context: '<input class="MuiInputBase-input-192 MuiInput-input-177 HvInput-input-153 MuiInputBase-inputAdornedEnd-198" placeholder="Search" type="text" value="">',
      selector: '#root > div:nth-child(2) > div > div > div > div:nth-child(2) > input',
      runner: 'axe',
      runnerExtras: [Object]
    },
    {
      code: 'WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name',
      type: 'error',
      typeCode: 1,
      message: 'This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.',
      context: '<input class="MuiInputBase-input-192 MuiInput-input-177 HvInput-input-153 MuiInputBase-inputAdornedEnd-198" placeholder="Search" type="text" value="">',
      selector: '#root > div:nth-child(2) > div > div > div > div:nth-child(2) > input',
      runner: 'htmlcs',
      runnerExtras: {}
    },
    {
      code: 'WCAG2AA.Principle1.Guideline1_3.1_3_1.F68',
      type: 'error',
      typeCode: 1,
      message: 'This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.',
      context: '<input class="MuiInputBase-input-192 MuiInput-input-177 HvInput-input-153 MuiInputBase-inputAdornedEnd-198" placeholder="Search" type="text" value="">',
      selector: '#root > div:nth-child(2) > div > div > div > div:nth-child(2) > input',
      runner: 'htmlcs',
      runnerExtras: {}
    }
  ]
}
```
